resources/migrations/053_add_query_table.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: 53
+      author: camsaul
+      changes:
+        - createTable:
+            tableName: query
```

Comment:
my thinking was down the road if we wanted to store the actual query itself or other info about the query we could just add a column to this table